### ISSUE TITLE
Create a new CTSM branch for "NorESM_LandSites_Platform"

### ISSUE THREAD

**Issue by huitang-earth** [NorESMhub/CTSM#5]
This issue is related to the issue [205](https://github.com/NorESMhub/NorESM/issues/205) in NorESM repository and issue [20](https://github.com/NorESMhub/cime/issues/20) in CIME repository.

The new CTSM branch will be used to keep the newest version of CTSM from NCAR that works with the most recent version of other model component (e.g., FATES). Note that whether this branch works with the atmosphere and ocean model components of NorESM is not tested. 

The new CTSM branch will also contain a lot of setup for running single-cell simulations over observation sites.

The new CTSM branch is ready at [my local repository](https://github.com/huitang-earth/ctsm/tree/noresm_site_platform). I can make a pull request if we decide on the name of the new branch. 

What would be a proper name for the new CTSM branch? @annefou, @kjetilaas, @sunnivin, @DirkOlivie, @matsbn, @MichaelSchulzMETNO

**Comment by matsbn** [comment]
How different will the new CTSM branch be from the ESCOMP CTSM master? If it is very similar, will it make sense to fetch updates from the ESCOMP repository and update master and other key branches in our fork so that the new local CTSM branch stays reasonably close to the forked master? I guess this can be done at a later stage, but I think it is important to think through this. A similar question is relevant for the issue https://github.com/NorESMhub/cime/issues/20.

**Comment by kjetilaas** [comment]
Agreed, @matsbn. A new brach only makes sense if there are specific changes to CTSM for this platform. Otherwise, I agree that we should just keep master up to date. This is probably different than with cime, where there are in deed a lot of changes specific to this platform.

**Comment by huitang-earth** [comment]
@matsbn @kjetilaas Thanks for the comments. Yes, I am planning to separate the configuration or namelist changes from CTSM, so that we could directly use the master branch of CTSM. The configuration or namelist files will then be kept in "NorESM_LandSites_Platform" and copied from there to the corresponding folder during the running phase. We will try to do the same for CIME. Hopefully, it will work. :)

**Comment by annefou** [comment]
Awesome! 💯 

@huitang-earth  will the change  you made in src/utils/clmfates_interfaceMod.F90 be reported back to ESCOMP/CTSM (https://github.com/huitang-earth/ctsm/commit/ff424d1963b5c687ab5073567f4cac7558875804)?

**Comment by huitang-earth** [comment]
Yes. It has been reported back. But they are slow in remove the bug: https://github.com/ESCOMP/CTSM/issues/1248

**Comment by huitang-earth** [comment]
@matsbn @kjetilaas, I have taken out all the configuration changes for the platform in CTSM, and keep them as separate files in the "NorESM_LandSItes_Platform" repo. Please see the PR (https://github.com/NorESMhub/NorESM_LandSites_Platform/pull/2)

So, we don't need to create a new branch of CTSM and CIME for the land sites platform now. But do you want me to update the "master" branch of NorESMhub/CTSM to be syn with the ESCOMP/CTSM master? I can do it quickly if needed.

**Comment by MichaelSchulzMETNO** [comment]
It would be good to have a short description of all major branches in the NorESMhub/CTSM repo in a readme.md file, which is in the master branch on the CTSM repo.

I have recently added the branch release-cesm2.2 to have the CTSM for an update to CESM/NorESM2.2 . This should contain the latest necessary for CESM/NorESM2.2. The question might be what the master in the NorESMhub/CTSM repo shall be used for. I would suggest this is a discussion and decision for the NorESM-CTSM core group.

**Comment by huitang-earth** [comment]
I have the same concern on "**how the master in the NorESMhub/CTSM repo shall be used for**". It can be (1) the "master" branch that always syn with ESCOMP CTSM without any changes, (2) the "master" branch of CTSM used by NorESM, in which we will keep track of the version used by NorESM. Hope to have a more thorough discussion.

**Comment by MichaelSchulzMETNO** [comment]
I suggest that you have a CTSM core telecon, I can join in.

**Comment by kjetilaas** [comment]
> I suggest that you have a CTSM core telecon, I can join in.
Sounds like a good idea. I can send around a doodle for a meeting early after easter.

**Comment by huitang-earth** [comment]
Perhaps, we could also discuss how we are going to merge this pull request #6 for CTSM. Do we need to push these changes back to ESCOMP/CTSM?.  It seems to be useful for the wider CESM/CLM community.

**Comment by rosiealice** [comment]
@huitang-earth can we close this issue in the absence of any activity on the platform?
